src/components/layouts/Sidebar.js

```diff
@@ -3,36 +3,64 @@ import './layouts.style.css';
+          <Link to="/" style={{ textDecoration: 'none' }}>
+            <div>
+              <div className="sidebar-inner">
+                <img
+                  src="assets/icons/dashboard-icon.svg"
+                  className="sidebar-inner"
+                  alt="dashboard"
+                />
+                <div className="sidebar-head sidebar-inner">Dashboard</div>
               </div>
-            </Link>
-            <Link to='/monitoring' style={{ textDecoration: 'none' }}>
-              <div>
-                <div className='sidebar-inner'>
-                  <img src='assets/icons/monitoring-icon.svg' className='sidebar-inner' alt='monitoring' />
-                  <div className='sidebar-head sidebar-inner'>Monitoring</div>
-                </div>
+            </div>
+          </Link>
+          <Link to="/monitoring" style={{ textDecoration: 'none' }}>
+            <div>
+              <div className="sidebar-inner">
+                <img
+                  src="assets/icons/monitoring-icon.svg"
+                  className="sidebar-inner"
+                  alt="monitoring"
+                />
+                <div className="sidebar-head sidebar-inner">Monitoring</div>
               </div>
-            </Link>
-            <Link to='/benchmarks' style={{ textDecoration: 'none' }}>
-              <div>
-                <div className='sidebar-inner'>
-                  <img src='assets/icons/bench-icon.svg' className='sidebar-inner' alt='Benchmarks' />
-                  <div className='sidebar-head sidebar-inner'>Benchmarks</div>
-                </div>
+            </div>
+          </Link>
+          <Link to="/benchmarks" style={{ textDecoration: 'none' }}>
+            <div>
+              <div className="sidebar-inner">
+                <img src="assets/icons/bench-icon.svg" className="sidebar-inner" alt="Benchmarks" />
+                <div className="sidebar-head sidebar-inner">Benchmarks</div>
               </div>
-            </Link>
-          </div>
-        </Router>
-      );
-    }
-}
\ No newline at end of file
+            </div>
+          </Link>
+          <Link to="/monitoring" style={{ textDecoration: 'none' }}>
+            <div>
+              <div className="sidebar-inner">
+                <img
+                  src="assets/icons/monitoring-icon.svg"
+                  className="sidebar-inner"
+                  alt="monitoring"
+                />
+                <div className="sidebar-head sidebar-inner">Monitoring</div>
+              </div>
+            </div>
+          </Link>
+          <Link to="/benchmarks" style={{ textDecoration: 'none' }}>
+            <div>
+              <div className="sidebar-inner">
+                <img src="assets/icons/bench-icon.svg" className="sidebar-inner" alt="Benchmarks" />
+                <div className="sidebar-head sidebar-inner">Benchmarks</div>
+              </div>
+            </div>
+          </Link>
+        </div>
```

Comment:
Please fix this. The components- monitoring and benchmarks have been repeated here.

Reply:
Oops!, might be during resolving the merge conflicts.

Reply:
Done!